Sources/UXKit/AppKit/NSTableViewCell.swift

```diff
@@ -249,19 +374,20 @@
     // MARK: - Label Factory
     
     open func makeLabel() -> UXLabel {
-      let v = NSTextField(frame: UXRect())
-      v.translatesAutoresizingMaskIntoConstraints = false
-      
-      /* configure as label */
-      v.isEditable      = false
-      v.isBezeled       = false
-      v.drawsBackground = false
-      v.isSelectable    = false // not for raw labels
-      
-      /* common */
-      v.alignment   = .left
-      
-      return v
+        let v = UXLabel(frame: UXRect())
+        v.cell = VerticallyCenteredTextFieldCell()
```

Comment:
Is the centering of all axis really what UITableView does?
I'd also move the Cell implementation into this function and not make it public API.